modules/UI/UI.js

```diff
@@ -227,8 +231,8 @@ UI.initConference = function () {
 
     APP.store.dispatch(checkAutoEnableDesktopSharing());
 
-    if(!interfaceConfig.filmStripOnly) {
-        Feedback.init(eventEmitter);
+    if (interfaceConfig.filmStripOnly) {
+        APP.store.dispatch(shouldShowPostCallFeedbackDialog(false));
```

Comment:
Not sure about this one. We already check interfaceConfig.filmStripOnly on so many places in react, so we can just move this check directly to the initialising of the feedback button.